ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -0,0 +1,500 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Service Catalog: EC2 Reference Architecture.'
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Linux Instance Configuration
+        Parameters:
+          - EC2InstanceType
+          - LinuxDistribution
+          - VolumeSize
+          - PrivateNetwork
+    ParameterLabels:
+      EC2InstanceType:
+        default: EC2 Instance Type
+      NotebookType:
+        default: Notebook Type
+      VolumeSize:
+        default: Disk Size
+      PrivateNetwork:
+        default: Use private network?
+Mappings:
+  NotebookTypes:
+    Rstudio:
+      AMIID: ami-036688a77595e31cf
```

Comment:
maybe add a comment and refer to the build of this AMI?  https://travis-ci.org/Sage-Bionetworks/packer-rstudio/builds/658411244

Reply:
done